packages/loader/container-loader/src/container.ts

```diff
@@ -736,7 +736,7 @@ export class Container
 				// dmLastMsqSeqNumber: if present, same as dmLastProcessedSeqNumber
 				dmLastMsqSeqNumber: () => this.deltaManager?.lastMessage?.sequenceNumber,
 				dmLastMsqSeqTimestamp: () => this.deltaManager?.lastMessage?.timestamp,
-				dmLastMsqSeqClientId: () => this.deltaManager?.lastMessage?.clientId,
+				dmLastMsqSeqClientId: () => this.deltaManager?.lastMessage?.clientId ?? undefined,
```

Comment:
I'd probably leave this as `null` unless/until we actually change what the server sends, otherwise it invites confusion about whether undefined/null mean different things

Reply:
If we have to do something else, then probably every place we send it in telemetry we should be doing the same thing (maybe string `"null"` here too as suggested below).  Maybe also export some `telemetrySafeServerClientId` from a telemetry package to be used in these sorts of locations?